Cargo.toml

```diff
@@ -12,17 +12,16 @@ alloy-primitives = { version = "0.6", default-features = false, features = ["rlp
 alloy-sol-types = { version = "0.6" }
 anyhow = { version = "1.0.75" }
 bincode = { version = "1.3" }
-bonsai-sdk = { git = "https://github.com/risc0/risc0", branch = "main" }
 bytemuck = { version = "1.14" }
 ethers = { version = "2.0" }
 hex = { version = "0.4" }
 log = { version = "0.4" }
 methods = { path = "./methods" }
-risc0-build = { git = "https://github.com/risc0/risc0", branch = "main", features = ["docker"] }
-risc0-build-ethereum = { git = "https://github.com/risc0/risc0-ethereum", branch = "capossele/refactor-zkvm-usage" }
-risc0-ethereum-contracts = { git = "https://github.com/risc0/risc0-ethereum", branch = "capossele/refactor-zkvm-usage" }
-risc0-zkvm = { git = "https://github.com/risc0/risc0", branch = "main", default-features = false }
-risc0-zkp = { git = "https://github.com/risc0/risc0", branch = "main", default-features = false }
+risc0-build = { version = "1.0.0-rc.6", features = ["docker"] }
+risc0-build-ethereum = { git = "https://github.com/risc0/risc0-ethereum", branch = "victor/release-1.0" }
+risc0-ethereum-contracts = { git = "https://github.com/risc0/risc0-ethereum", branch = "victor/release-1.0" }
+risc0-zkvm = { version = "1.0.0-rc.6", default-features = false }
+risc0-zkp = { version = "1.0.0-rc.6", default-features = false }
```

Comment:
I thought this branch (`dev` eventually) is supposed to use main rather than a specific release. Or is this only an exercise to test that all is working?